src/test/json/events_non_finite_to.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2019-2020 Dr. Colin Hirsch and Daniel Frey
+// Please see LICENSE for license or visit https://github.com/taocpp/json/
+
+#include "test.hpp"
+#include "test_types.hpp"
+
+#include <tao/json/events/non_finite_to_exception.hpp>
+#include <tao/json/events/non_finite_to_null.hpp>
+#include <tao/json/events/non_finite_to_string.hpp>
+
+#include <tao/json.hpp>
+
+namespace tao::json
+{
+   value test_non_finite_to_string( const value& v, const std::string& s )
+   {
+      return s.compare( to_string< events::non_finite_to_string >( v ) );
```

Comment:
Why not `to_string<...>(v) == s`? This is C++, not Java ;)

Reply:
I'll make the change! gotta get out of old habits haha.